setup.py

```diff
@@ -328,7 +328,7 @@ def finalize_options(self):
     def run(self):
         subprocess.check_call("mkdir -p _gh-pages/latest && touch _gh-pages/.nojekyll && "
                               "cd docs; cp Makefile.gh_pages Makefile; cp conf_std.py conf.py; sphinx-apidoc "
-                              " -o ./ -f -A 'PyCBC dev team' -V '0.1' ../pycbc && make html",
+                              " -o ./ -f -A 'PyCBC dev team' -V '0.1' ../pycbc ../pycbc/ligolw && make html",
```

Comment:
What's the point of this? Is this building documentation for ligolw? If this is temporary, probably best not to expose this, as hopefully, ligolw will be used from normal glue or not all, no?

Reply:
This explicitly turns off documenting the ligolw module (the syntax is strange). The documentation would fail anyway as the docstrings for functions are not in sphinx formatting.

This change is needed.